modules/CmpdReg/src/client/templates/LotForm/ParentView.inc

```diff
@@ -1,8 +1,8 @@
 <script type="text/template" id="LotForm_ParentView_template">
 
     <div class="radioWrapper">
-        <input type="radio" name="regPick" class="regPick" ></input>
-        <label>New <span class="lotOrBatch"></span> <span class="corpName"></span></label>
+        <input id="newLot" type="radio" name="regPick" class="regPick" ></input>
+        <label for="newLot">New <span class="lotOrBatch"></span> <span class="corpName"></span></label>
```

Comment:
Unrelated issue but there was a set of radio buttons on Reg search to select a found structure or create a new one and you couldn't click the label.